CHANGELOG.md

```
@@ -0,0 +1,51 @@
+# Changelog
+
+All notable changes to FORGE are documented in this file.
+
+## [0.2.0] — 2026-02-19
+
+### Phase 2 Features
+
+#### WI-26 — XML Output
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)